ui/typedoc.json

```diff
@@ -1,7 +1,12 @@
 {
   "entryPointStrategy": "packages",
   "entryPoints": [
-    "."
```

Comment:
Explicitly including the non-private packages here because otherwise you get weird empty entries in typedoc for `app` and some of the internal packages. 

At some point, we may want to reorganize the packages a little bit to make it easier to handle private vs. public packages in code like this and some of our release utils, but I don't want to tackle it in this particular PR.